src/blob/index/bptree/core.rs

```diff
@@ -0,0 +1,283 @@
+/// structure of b+-tree index file from the beginning:
+/// 1. Header
+/// 2. Bloom filter
+/// 3. Sorted by keys array of record headers (serialized size of one record header is 61 bytes)
+/// 4. Metadata
+/// 5. Sorted by keys array of pairs `(key, offset of record header in file)` (these are leaf nodes
```

Comment:
Are there any consistency control on block level? I saw hash for header.

Reply:
There is consistency control on the whole file level. I think, that's enough to be sure that every block is consistent (but maybe I misunderstood you)